cli/src/deploy/mod.rs

```diff
@@ -131,15 +157,32 @@ pub async fn add_cred() -> String {
   let name = cred_name.to_string();
   match cloud {
     "AWS" => {
-      // TODO check ~/.aws/credentials and provide values as default
-      let access_key: String = Input::with_theme(&ColorfulTheme::default())
-        .with_prompt("AWS Access Key ID")
-        .interact_text()
-        .unwrap();
-      let secret: String = Input::with_theme(&ColorfulTheme::default())
-        .with_prompt("AWS Secret Access Key")
-        .interact_text()
-        .unwrap();
+      let (access_key, secret) = match get_aws_cli_creds() {
+        Ok(aws_cli_creds) => {
```

Comment:
*Maybe* we should have a message on the screen saying that we found credentials for their account in this case?

Just having their current credentials without an explanation might be alarming.